src/components/TeamForm/TeamForm.js

```diff
@@ -28,20 +30,26 @@ class TeamForm extends Component {
     };
     this.state = {
       address: props.team.address,
-      name: props.team.name
+      name: props.team.name,
+      voteDuration: props.team.vote_duration
     };
   }
 
   handleChange = field => event => this.setState({ [field]: event.target.value });
 
+  handleKeyPress = field => event => field === 'voteDuration' ? onlyAllowNonZeroDigits(event) : null;
+
   handleSubmit = (event) => {
+    // TODO: add error if null value is submitted for vote duration and prevent submission
     event.preventDefault();
     this.props.updateTeam(Object.assign({}, this.state, this.props.center));
   };
 
   render() {
     const { team } = this.props;
     const { name } = this.state;
+    const voteDuration = this.state.voteDuration;
+    const voteDurationLabel = voteDuration === 1 ? 'day' : 'days';
```

Comment:
Will probably leave off the `voteDurationLabel` entirely as part of the refactor, and just include a one or two sentence description of what the `Vote Duration` means.

Reply:
Sure, I think a description here would be very helpful. The term "vote duration" might itself be a little ambiguous, as votes themselves are displayed for only 12 hours until they only affect future sorting. Perhaps the label (and help text) for this form field could be "Sort duration" instead (since decisions affect sorting in addition to votes)?

Reply:
Yes, this makes sense.